src/Test.FunctionalTests.BinSkim.Driver/AnalyzeCommandTests.cs

```diff
@@ -135,6 +135,58 @@ public void AnalyzeCommand_DeterminismTest()
             log.Runs[0].Invocations[0].ToolConfigurationNotifications.Count(t => t.Message.Text.Contains("E_PDB_NOT_FOUND")).Should().Be(1);
         }
 
+        [Fact]
+        public void AnalyzeCommand_IgnoreExceptionInCanAnalyzeErrorTest()
```

Comment:
Do you think reflecting this test is specific for PEs makes sense in test name here?